src/policies/Termsofservice.tsx

```
@@ -0,0 +1,17 @@
+
+import Footer from '../components/Footer';
+import Navbar from '../Navbar';
+import TermsOfService from './TOS Component';
+
+function TOS() {
+  return (
+    <div>
+        <Navbar />;
+    <TermsOfService />
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)